resources/assets/coffee/react/beatmap-discussions/discussion.coffee

```diff
@@ -166,8 +168,8 @@ class BeatmapDiscussions.Discussion extends React.PureComponent
       user: @props.users[post.user_id]
       lastEditor: @props.users[post.last_editor_id]
       canBeEdited: @props.currentUser.isAdmin || @isOwner(post)
```

Comment:
seems to need update as well

Reply:
whoops, they shouldn't be able to edit posts